### PR TITLE
Add musl build for aarch64 and armv7 binaries

### LINKED ISSUE #712: armv7 binaries are not available
Raspberry pi 3 and others are still being sold and available.

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #712 

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -50,6 +50,15 @@ jobs:
           - build: linux-arm
             os: ubuntu-22.04
             target: aarch64-unknown-linux-gnu
+          - build: linux-arm
```
Comment (fujiapple852) [inline .github/workflows/release.yml:53]: @patrickelectric I believe each build name has to be unique included in the matrix build list. I suggest renaming the Linux build jobs to include the architecture & gnu/musl information.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -50,6 +50,15 @@ jobs:
...
+            os: ubuntu-22.04
+            target: aarch64-unknown-linux-musl
+          - build: linux-armv7
+            os: ubuntu-22.04
+            target: arm-unknown-linux-gnueabi
```
Comment (fujiapple852) [inline .github/workflows/release.yml:58]: @patrickelectric there seem to be armv7 specific targets (https://doc.rust-lang.org/rustc/platform-support.html), should they be used here instead?

**Comment by fujiapple852** [comment]
Thanks for the PR @patrickelectric !

I’ve added a couple of review comments. 

Once resolved i’d like to see this run on your fork to teat it out (unfortunately we can’t test it here without doing an actual release!). 

The way to do that is to merge it to master on your fork and add a dummy release tag (1.2.3 is fine) to trigger the GitHub release workflow.

**Comment by c-git** [comment]
ouch, at least the changes aren't lost

**Comment by patrickelectric** [comment]
Ouch I deleted the repository by mistake, I forgot about the PR! Will check it soon.

**Comment by patrickelectric** [comment]
Ouch I deleted the repository by mistake, I forgot about the PR! Will check it soon.

**Comment by c-git** [comment]
Yeah I suspected, at least github didn't delete all of your work.

**Pull request closed without merging**